src/modal/modal.spec.ts

```diff
@@ -876,12 +876,6 @@ export class WithSkipTabindexFirstFocusableModalCmpt {
       </ng-template>
     </ng-template>
     <button id="open" (click)="open('from button')">Open</button>
-    <div id="open-no-focus" (click)="open('from non focusable element')">Open</div>
```

Comment:
Any reason for removing this part of the test?

Reply:
This part does not pass linter.

It does not affect CI, so removing it completely seems better than fixing it.